spec/acceptance/basic_spec.rb

```diff
@@ -117,3 +117,54 @@ class { 'pulpcore':
   end
 
 end
+
+describe 'with content cache enabled' do
+  certdir = '/etc/pulpcore-certs'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      class { 'pulpcore':
+        cache_enabled => true,
+      }
+      PUPPET
+    end
+  end
+
+  describe service('httpd') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-content') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe port(80) do
+    it { is_expected.to be_listening }
+  end
+
+  describe port(443) do
+    it { is_expected.to be_listening }
+  end
+
+  describe port(6379) do
+    it { is_expected.to be_listening }
+  end
+
+  describe service('rh-redis5-redis'), unless: %w[centos redhat].include?(os[:family]) && os[:release].to_i == 8 do
+    it { is_expected.to be_running }
+    it { is_expected.to be_enabled }
+  end
+
+  describe service('redis'), unless: %w[centos redhat].include?(os[:family]) && os[:release].to_i == 7 do
```

Comment:
What I meant in https://github.com/theforeman/puppet-pulpcore/pull/204#discussion_r663782249 was that one entry would be `if` and the other `unless` with the same condition in both places.
```suggestion
  describe service('rh-redis5-redis'), if: %w[centos redhat].include?(os[:family]) && os[:release].to_i == 7 do
    it { is_expected.to be_running }
    it { is_expected.to be_enabled }
  end

  describe service('redis'), unless: %w[centos redhat].include?(os[:family]) && os[:release].to_i == 7 do
```